tests/test_custom_checkers/test_forbidden_python_syntax_checker.py

```diff
@@ -17,105 +18,57 @@ def set_up(self) -> None:
         """Perform the set up before each test case executes."""
         self.setup_method()
 
-    def test_disallow_break_in_code(self) -> None:
-        """Test that the checker correctly reports a break statement in the code when its usage is
-        disallowed.
-        """
-        src = """
-        for i in range(0, 10):
-            break
-        """
-        mod = astroid.parse(src)
-        break_node, *_ = mod.nodes_of_class(nodes.Break)
-        name = break_node.__class__.__name__
-
-        with self.assertAddsMessages(
-            pylint.testutils.MessageTest(
-                msg_id="forbidden-python-syntax", node=break_node, args=name
+    @pytest.mark.parametrize(
+        "src, expected_node_type",
+        [
+            (
+                """
+            for i in range(0, 10):
```

Comment:
The indentation here is strange; the text and closing `"""` should be aligned with the opening `"""`